37/generated/admin_client_config.html

```diff
@@ -284,7 +284,7 @@ <h4><a id="ssl.enabled.protocols"></a><a id="adminclientconfigs_ssl.enabled.prot
 <p>The list of protocols enabled for SSL connections. The default is 'TLSv1.2,TLSv1.3' when running with Java 11 or newer, 'TLSv1.2' otherwise. With the default value for Java 11, clients and servers will prefer TLSv1.3 if both support it and fallback to TLSv1.2 otherwise (assuming both support at least TLSv1.2). This default should be fine for most cases. Also see the config documentation for `ssl.protocol`.</p>
 <table><tbody>
 <tr><th>Type:</th><td>list</td></tr>
-<tr><th>Default:</th><td>TLSv1.2,TLSv1.3</td></tr>
```

Comment:
I'm not sure we want these TLS config changes. It looks like the docs were generated with Java 8.

Reply:
Thanks for poiting this out. I'm not sure how this happened. I had another look to make the script is [using Java 17](https://github.com/apache/kafka/pull/16287/files#diff-241bcfc06fc20c009b77012a614513cc634006b8557f4d9cdcdc8fb8d44178e6R135).  Regenerated site docs, javadoc and rebased.